pipelines/batch/src/test/java/org/openmrs/analytics/ConvertResourceFnTest.java

```diff
@@ -135,7 +167,13 @@ public void testResourceMetaTags() throws IOException, java.text.ParseException,
         Resources.toString(Resources.getResource("patient.json"), StandardCharsets.UTF_8);
     HapiRowDescriptor element =
         HapiRowDescriptor.create(
-            "123", "Patient", "2020-09-19 12:09:23", "R4", "1", patientResourceStr);
+            "123",
+            "forced-id-123",
```

Comment:
Question: Why in this case the `capturedResource.getId()` is not changed? I mean why don't you need to update line 192 below to `forced-id-123` instead of `123`?

Reply:
Thanks @bashir2 for the pointer. It should be `force-id-123`. The `@Test` annotation was missing for this test case.